### PR TITLE
Improve eslint config

### PR DESCRIPTION
**Pull request by nicolaserny**
- Use Vue recommended rules (instead of essential).
- Use @vue/airbnb typescript  rules (standard rules provided by Airbnb)

### CODE REVIEW

File: viv-wallet-app/src/components/BalanceCard.vue [hunk 1]
```diff
@@ -15,7 +15,7 @@
 import { defineComponent } from "vue";
 
 export default defineComponent({
-    name: "balande-card",
+    name: "BalandeCard",
```
Comment (maurelio1234) [inline viv-wallet-app/src/components/BalanceCard.vue:18]: typo? Balan*c*e card?

File: viv-wallet-app/src/services/login.ts [hunk 2]
```diff
@@ -1,3 +1,4 @@
+// eslint-disable-next-line camelcase
```
Comment (maurelio1234) [inline viv-wallet-app/src/services/login.ts:1]: can we rename when we import? it avoids the `disable`...

File: viv-wallet-app/src/views/Payment.vue [hunk 3]
```diff
@@ -77,25 +75,39 @@ export default defineComponent({
...
         },
     },
+    async mounted() {
+        try {
+            [this.user, this.balance, this.unpaidActions] = await Promise.all([
+                this.usersService.getUser(this.id),
+                this.walletService.getUserBalance(this.id),
+                this.walletService.getUserPayableActions(this.id),
+            ]);
+            this.viv += this.balance;
```
Comment (maurelio1234) [inline viv-wallet-app/src/views/Payment.vue:92]: it's weird that we do that on the front end anywat... just a comment

File: viv-wallet-app/src/views/Wallet.vue [hunk 4]
```diff
@@ -100,6 +89,17 @@ export default defineComponent({
             },
         },
     },
+    methods: {
+        formatConsultantStatus(status?: keyof typeof ConsultantStatus) {
+            if (status) {
+                return toString(ConsultantStatus[status]);
+            }
+            return "";
```
Comment (maurelio1234) [inline viv-wallet-app/src/views/Wallet.vue:97]: maybe console warn ?

**Review by maurelio1234 — CHANGES_REQUESTED** [review]

File: viv-wallet-app/src/views/Payment.vue (hunk 3 above)
Comment (nicolaserny) [reply viv-wallet-app/src/views/Payment.vue:92]: Yes, I know, but this part will change soon. We're working on it with Benoit.

File: viv-wallet-app/src/components/BalanceCard.vue (hunk 1 above)
Comment (nicolaserny) [reply viv-wallet-app/src/components/BalanceCard.vue]: done

File: viv-wallet-app/src/services/login.ts (hunk 2 above)
Comment (nicolaserny) [reply viv-wallet-app/src/services/login.ts]: I don't even know why I did not do that in the first place. 🤣

File: viv-wallet-app/src/views/Wallet.vue (hunk 4 above)
Comment (nicolaserny) [reply viv-wallet-app/src/views/Wallet.vue:98]: done

Comment (maurelio1234) [reply viv-wallet-app/src/views/Wallet.vue:98]: nice, but i think there's nothing interesting in `status`. but why not :)

**Review by maurelio1234 — APPROVED** [review]

**Pull request merged**